src/tools/gxhttpd.ss

```diff
@@ -281,8 +283,10 @@
          (mux (make-mux cfg))
          (request-logger (get-request-logger cfg))
          (addresses (config-get! cfg listen:))
+	 (max-token-length (config-get cfg max-token-length: 1024))
          (run-httpd
           (lambda ()
+	    (set-httpd-max-token-length! max-token-length)
```

Comment:
we should type check that, so wrap a runtime check with (: ... :fixnum).

Maybe at parse site.

Reply:
I added it at server execution run time as ... heh ... I'm working on parsers and in this case just because the value passed to the server should be a fixnum does not mean the parser needs that constraint.